plugins/woocommerce-blocks/tests/e2e/tests/product-collection/product-collection.block_theme.spec.ts

```diff
@@ -1074,6 +1074,59 @@ test.describe( 'Editor product picker', () => {
 				);
 			} );
 
+			test( `For collection "${ collection.name }" - changing product using inspector control`, async ( {
+				pageObject,
+				admin,
+				page,
+			} ) => {
+				await admin.createNewPost();
+				await pageObject.insertProductCollection();
+				await pageObject.chooseCollectionInPost( key as Collections );
```

Comment:
You can use a helper instead:

```suggestion
				await pageObject.createNewPostAndInsertBlock( key )
```

Reply:
I didn't use `createNewPostAndInsertBlock` because I added a `chooseProductInEditorProductPickerIfAvailable` call in this function, which chooses a product in the product picker. In this specific test, we don't want that to run 🙂